rpcs3/Emu/Cell/Modules/cellGame.cpp

```diff
@@ -802,8 +802,8 @@ error_code cellGameDataCheckCreate2(ppu_thread& ppu, u32 version, vm::cptr<char>
 		strcpy_trunc(cbGet->getParam.titleLang[i], psf::get_string(sfo, fmt::format("TITLE_%02d", i)));
 	}
 
-	std::memset(g_file_param.get_ptr(), 0, sizeof(*g_file_param));
-	cbSet->setParam = g_file_param;
+	//std::memset(g_file_param.get_ptr(), 0, sizeof(*g_file_param));
```

Comment:
Please add a comment

Reply:
Fixed.

Reply:
the setParam is assigned in the CELL_GAMEDATA_CBRESULT_OK below.
Maybe just move this code there

Reply:
I think you can safely remove this code.
The developer is supposed to set this parameter in funcStat, which is then used to update the PARAM.SFO.
It's recommended to use null, since it's some broken legacy feature.